includes/admin/server-timing.php

```diff
@@ -35,14 +37,16 @@ function perflab_add_server_timing_page() {
 
 	return $hook_suffix;
 }
+
+// @phpstan-ignore-next-line
```

Comment:
Because PHPStan identifies correctly that this function is used as an action, and not a filter. So it shouldn't return anything.